src/create-card.js

```diff
@@ -304,7 +326,7 @@ class CardCreator {
    * });
    * @returns {Promise<Buffer>} A promise representating the construction of the card's image data.
    */
-  async createCard(characterId, customImage, language = 'en') {
+  async createCard(characterId, customImage= null, language = 'en') {
```

Comment:
needs space?

Reply:
I agree. I might wait on style fixes for now though as I'm going to break out the files and convert to ts after this.